Dev/src/components/TopBanner.js

```diff
@@ -8,11 +8,12 @@ import { Icon } from 'react-native-elements';
 import colors from 'config/colors'
 import fontStyles from 'config/fontStyles';
 import { screenHeight, screenWidth } from 'config/dimensions';
+import { TextInput } from 'react-native-gesture-handler';
```

Comment:
This is the wrong TextInput. Use the react-native version. "import { TextInput } from 'react-native'.

Reply:
wdym its the wrong textinput